apps/juicebox_web/test/lib/web/channels/stream_channel_test.exs

```diff
@@ -76,4 +76,56 @@ defmodule JuiceboxWeb.StreamChannelTest do
       assert_broadcast "remote.action", payload
     end
   end
+
+  describe "Connecting" do
+    setup [:with_some_reactions, :with_some_videos, :connect]
+
+    #test "pushes all current reactions", %{socket: socket} do
+    #  assert_push("remote.action", %{video: "video 1", user_id: "user 1", "type": "NEW_REACTION"})
+    #  assert_push("remote.action", %{video: "video 2", user_id: "user 2", "type": "NEW_REACTION"})
+    #  assert_push("remote.action", %{video: "video 3", user_id: "user 3", "type": "NEW_REACTION"})
+    #end
+
+    test "pushes the current queue" do
+      assert_push("remote.action", %{videos: [%{video: @vide}, %{video: @video_2}], "type": "QUEUE_UPDATED" })
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.